src/store/connection/reducer.js

```diff
@@ -76,7 +76,7 @@ export default function connectionsReducer(state = initialState, action) {
     case APPEND_CONNECTIONS:
       return {
         ...state,
-        connections: [...state.connections, ...action.payload],
+        connections: [...state.connections, action.payload],
```

Comment:
I believe we should add new conenction to the start of list: `connections: [ action.payload, ...state.connections],`